pallets/asset-manager/src/mock.rs

```diff
@@ -240,14 +261,16 @@ pub(crate) fn create_asset_metadata(
     evm_address: Option<H160>,
     is_frozen: bool,
     is_sufficient: bool,
-) -> AssetRegistrarMetadata {
-    AssetRegistrarMetadata {
-        name: name.as_bytes().to_vec(),
-        symbol: symbol.as_bytes().to_vec(),
-        decimals,
+) -> AssetRegistryMetadata<Balance> {
+    AssetRegistryMetadata {
+        metadata: AssetStorageMetadata {
+            name: name.as_bytes().to_vec(),
+            symbol: symbol.as_bytes().to_vec(),
+            decimals,
+            is_frozen,
```

Comment:
1. `AssetRegistrarMetadata` is renamed to `AssetRegistryMetadata`, so does old storage need to be removed from here?: https://github.com/Manta-Network/Manta/pull/814/files#diff-40225a2ea5f6a026936004b2aab2d0126e1f4c481c089528d70f81b819c507e9L195 
2. Because the filed `is_frozen` is moved to `metadata`, we might need do migration for `AssetRegistryMetadata`.

Reply:
good catch
1. This one is not a storage item. It needs to be annotated with `storage` to get a storage key https://github.com/paritytech/substrate/blob/master/frame/support/src/lib.rs#L215-L233
2. I would've assumed so, but actually executing this migration locally manages to migrate the `AssetRegistryMetadata`, which doesn't really make sense to me ...
But anyway I'll make this migration explicit in the code.

Reply:
Right, I migrated it -> https://github.com/Manta-Network/Manta/pull/814/files#diff-e29c5b8bac12f21073b76e6b3971b09b03d1c986c9918c8be66b581f4ca5cb17R171-R202

Reply:
Or do you mean something else ?

Reply:
Sorry, I didn't see it.